### PR TITLE
update profile test dir in e2e test

### PR DESCRIPTION
**Pull request by kunmingg**
following https://github.com/kubeflow/kubeflow/pull/4633

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfctl/165)

### CODE REVIEW

File: py/kubeflow/kfctl/testing/ci/kfctl_e2e_workflow.py [hunk 1]
```diff
@@ -336,7 +336,6 @@ def _build_step(self, name, workflow, dag_name, task_template,
     for t in workflow["spec"]["templates"]:
       if t["name"] == name:
         return t
-    workflow["spec"]["templates"].append(new_template)
```
Comment (jlewi) [inline py/kubeflow/kfctl/testing/ci/kfctl_e2e_workflow.py]: Why did this line get deleted?

Comment (kunmingg) [reply py/kubeflow/kfctl/testing/ci/kfctl_e2e_workflow.py]: Seems `new_template ` is not defined anywhere.

Comment (jlewi) [reply py/kubeflow/kfctl/testing/ci/kfctl_e2e_workflow.py]: Thanks. It looks like this is legacy code and this is now handled by 
https://github.com/kubeflow/testing/blob/f5aaa62db495a78816ac31a9673c3a0ba0602c8f/py/kubeflow/testing/argo_build_util.py#L151

Can we delete everything after line 333 and just do

```
return argo_build_util.add_task_to_dag(....)
```

Comment (kunmingg) [reply py/kubeflow/kfctl/testing/ci/kfctl_e2e_workflow.py]: updated

**Pull request merged**